index.html

```diff
@@ -13,18 +13,22 @@
         <div id="my-logo">
             <a href="#">CLARENCE</a>
         </div>
-        <nav class="menu-bar">
-            <img src="images/menu.svg" alt="menu bar">
+        <div class="hamberger-container" id="hamberger">
+            <span class="btn-line"></span>
+            <span class="btn-line"></span>
+            <span class="btn-line"></span>
+        </div>
+        <nav class="menu-bar" id="menu">
             <ul class="social-media">
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
-                <li><a href="#"><img src="images/email.png" alt="">
+                <li><a class="nav-links" href="#Portfolio">Portfolio</a></li>
+                <li><a class="nav-links" href="#about">About</a></li>
+                <li><a class="nav-links" href="#contact">Contact</a></li>
+                <li><a class="nav-links" href="#"><img src="images/email.png" alt="">
                 </a></li>
```

Comment:
- [ ] Nice work implementing the mobile menu :+1: However it would be nice to stick to the design as much as possible. The font and the color are not matching the Figma design and also kindly add a line under each option in the mobile menu as in the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-23 08-06-09](https://github.com/ClarenceAtim/Portfolio/assets/91142432/ff0d7f3c-f669-413b-8002-84f311bfd1e7)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-23 08-05-08](https://github.com/ClarenceAtim/Portfolio/assets/91142432/9d72c4a8-e274-4e09-9f59-e97bee11b812)